backend/controllers/userController.js

```diff
@@ -87,7 +88,15 @@ const updateUser = async (req, res) => {
 
   await user.save();
 
-  res.sendStatus(200);
+  if (language) {
+    const userForToken = {
+      id: userId,
+      lang: language,
+    };
+    user.token = jwt.sign(userForToken, process.env.SECRET);
+  }
+
+  res.json(user);
```

Comment:
maybe better send just token? in postman I receive all information about user that saved in mongo db. For example: 
```
if (language) {
    return res.json({token: jwt.sign(....)}) // something like that ?
}
```

Reply:
You are right! Fixed this and also that updating only avatar thing you spotted earlier :)